projects/admin/src/app/service/record-routing.service.spec.ts

```diff
@@ -15,26 +15,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { async, TestBed } from '@angular/core/testing';
+import { TestBed } from '@angular/core/testing';
 
-import { MylibraryComponent } from './mylibrary.component';
+import { RecordRoutingService } from './record-routing.service';
+// import { TranslateModule } from '@ngx-translate/core';
 import { RouterTestingModule } from '@angular/router/testing';
```

Comment:
I suggest to group import from third party libraries and local import ones.

Ex: 

```typescript
import { Injectable } from '@angular/core';
import { map, mergeMap, delay } from 'rxjs/operators';
import { of, Observable } from 'rxjs';

import { ToastrService } from 'ngx-toastr';
import { TranslateService } from '@ngx-translate/core';

import { DialogService } from '../dialog/dialog.service';
import { RecordService } from './record.service';
```